mindmeld/models/tagger_models.py

```diff
@@ -28,9 +28,14 @@
 )
 from .model import EntityModelEvaluation, EvaluatedExample, Model, ModelConfig
 from .taggers.crf import ConditionalRandomFields
-from .taggers.lstm import LstmModel
 from .taggers.memm import MemmModel
 
+try:
+    from .taggers.lstm import LstmModel
+except ImportError:
+    LstmModel = None
```

Comment:
Could we add a comment here explaining why we are doing this, ie, the tensorflow lib the LSTM model deps on is optional?